src/utils/snowflake.ts

```diff
@@ -9,20 +10,20 @@ export class Snowflake {
     return BigInt.asUintN(64, BigInt(this.id))
   }
 
-  get timestamp(): string {
-    return ((this.snowflake >> 22n) + 1420070400000n).toString()
+  get timestamp(): number {
+    return Number(((this.snowflake >> 22n) + 1420070400000n).toString())
```

Comment:
You don't need to change to string if I'm right

Reply:
Fixed.